### PR TITLE
target, optimization, animfix

### PR DESCRIPTION
**Pull request by i-kulgu**
**Describe Pull request**
- Added target when convar usetarget = true
- Fixed the animations
- Placed multiple parking spots to spawn the truck, it will spawn in other spot when the one is occupied
- Fixed the issue for getting the route again when you are busy with it, after finished the route you can get another route or get your paycheck
- Cleared unused code
- Updated language files

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes] Both target and drawtext are tested one day long
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

**Comment by sinphoenix** [comment]
please fixed the Lint issues main:1, main:2

**Comment by i-kulgu** [comment]
> please fixed the Lint issues main:1, main:2

Done

File: config.lua [hunk 1]
```diff
@@ -48,7 +48,10 @@ Config.Locations = {
     },
     ["vehicle"] = {
         label = "Garbage Truck Storage",
-        coords = vector4(-313.84, -1522.82, 27.56, 258.62),
+        coords = { -- parking spot locations to spawn garbage
+                [1] = vector4(-333.84, -1527.28, 27.28, 1.97),
+                [2] = vector4(-327.55, -1527.69, 27.25, 359.43),
+                },
```
Comment (D4isDAVID) [inline config.lua]:
```suggestion
            [1] = vector4(-333.84, -1527.28, 27.28, 1.97),
            [2] = vector4(-327.55, -1527.69, 27.25, 359.43),
        },
```
Fix the indentation here

File: server/main.lua [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 local Routes = {}
 
-QBCore.Functions.CreateCallback("garbagejob:server:NewShift", function(source, cb)
+local function CanPay(Player)
+    local bank = Player.PlayerData.money['bank']
+    if bank >= Config.TruckPrice then
+        return true
+    else
+        return false
+    end
```
Comment (D4isDAVID) [inline server/main.lua]:
```suggestion
    return Player.PlayerData.money['bank'] >= Config.TruckPrice
```
Code can be shortened here

File: client/main.lua [hunk 3]
```diff
@@ -144,6 +114,84 @@ local function DeliverAnim()
...
+            exports['qb-target']:AddCircleZone('garbagebin', vector3(CL.coords.x, CL.coords.y, CL.coords.z), 2.0,{
+                name = 'garbagebin', debugPoly = false, useZ=true}, {
+                options = {{label = Lang:t("target.grab_garbage"),icon = 'fa-solid fa-trash', action = function() TakeAnim() end}},
+                distance = 2.0
+            })
+        end
+    end
+end
+
+function TakeAnim()
```
Comment (D4isDAVID) [inline client/main.lua:163]: Lots of indentation problems in this function (`TakeAnim`), please convert the tabs to spaces.

File: client/main.lua [hunk 4]
```diff
@@ -395,72 +442,76 @@ RegisterNetEvent('garbagejob:client:SetWaypointHome', function()
...
+                end
+                if occupied then
+                    QBCore.Functions.Notify(Lang:t("error.all_occupied"))
+                end
+            end
+            currentStop = firstStop
+            currentStopNum = 1
+            amountOfBags = totalBags
+            print(currentStop)
+            print(amountOfBags)
```
Comment (D4isDAVID) [inline client/main.lua]:
```suggestion
```
Probably shouldn't be here

File: client/main.lua [hunk 5]
```diff
@@ -395,72 +442,76 @@ RegisterNetEvent('garbagejob:client:SetWaypointHome', function()
...
+                local occupied = false
+                local coords = Config.Locations["vehicle"].coords
+                for _,v in pairs(Config.Locations["vehicle"].coords) do
+                    if not IsAnyVehicleNearPoint(vector3(v.x,v.y,v.z), 2.5) then
+                        QBCore.Functions.TriggerCallback('QBCore:Server:SpawnVehicle', function(netId)
+                            local veh = NetToVeh(netId)
+                            SetVehicleEngineOn(veh, false, true)
+                            garbageVehicle = veh
+                            SetVehicleNumberPlateText(veh, "QB-" .. tostring(math.random(1000, 9999)))
+                            SetEntityHeading(veh, coords.w)
```
Comment (D4isDAVID) [inline client/main.lua]:
```suggestion
                            SetEntityHeading(veh, v.w)
```
Pretty sure it should be this?

**Review by D4isDAVID — CHANGES_REQUESTED** [review]
Seems to work fine, but the code needs to be sorted out

File: client/main.lua (hunk 3 above)
Comment (i-kulgu) [reply client/main.lua:163]: fixed the identation

Comment (D4isDAVID) [reply client/main.lua:163]: Not yet fixed. The rest of the file uses 4 spaces for indentation, your function uses tabs. Please convert the tabs to 4 spaces.

Comment (D4isDAVID) [inline client/main.lua:163]: Indentation here is not yet fixed. Some parts of your function use tabs, while the rest of the file uses 4 spaces for indentation. Please convert the tabs in your function to 4 spaces.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

**Review by D4isDAVID — APPROVED** [review]
LGTM

**Pull request merged**